moto/eks/responses.py

```diff
@@ -0,0 +1,124 @@
+from __future__ import unicode_literals
+from moto.core.responses import BaseResponse
+from .models import eks_backends
+import json
+
+DEFAULT_MAX_RESULTS = 100
+DEFAULT_NEXT_TOKEN = ""
+
+
+class EKSResponse(BaseResponse):
+    SERVICE_NAME = "eks"
+
+    @property
+    def eks_backend(self):
+        return eks_backends[self.region]
+
+    def create_cluster(self):
+        name = self._get_param("name")
+        version = self._get_param("version")
+        role_arn = self._get_param("roleArn")
+        resources_vpc_config = self._get_param("resourcesVpcConfig")
+        kubernetes_network_config = self._get_param("kubernetesNetworkConfig")
+        logging = self._get_param("logging")
+        client_request_token = self._get_param("clientRequestToken")
+        tags = self._get_param("tags")
+        encryption_config = self._get_param("encryptionConfig")
+
+        cluster = self.eks_backend.create_cluster(
+            name=name,
+            version=version,
+            role_arn=role_arn,
+            resources_vpc_config=resources_vpc_config,
+            kubernetes_network_config=kubernetes_network_config,
+            logging=logging,
+            client_request_token=client_request_token,
+            tags=tags,
+            encryption_config=encryption_config,
+        )
+
+        return 200, {}, json.dumps({"cluster": {**dict(cluster)}})
```

Comment:
The `**dict` notation doesn't work in Py2.7, as far as I know.
Other services will typically use a `model.to_dict()` method  to control the response format

Reply:
I get your comment about the **dict() usage in the responses file, and implemented a to_dict() in the Cluster and ManagedNodegroup models.  I've set up a venv with 2.7 top make sure I catch that stuff in the future.